File: server/iih_pub/iih.ci_pub/src/main/java/iih/ci/mrqc/itmdto/d/ItmFstDTO.java
package iih.ci.mrqc.itmdto.d;

import xap.mw.core.data.*;
import xap.mw.coreitf.d.*;
import java.math.BigDecimal;

/**
 * 质控项上下DTO DTO数据 
 * 
 */
public class ItmFstDTO extends BaseDTO {
	private static final long serialVersionUID = 1L;
	
	/**
	 * 一级分类
	 * @return String
	 */
	public String getId_fst() {
		return ((String) getAttrVal("Id_fst"));
	}
	/**
	 * 一级分类
	 * @param Id_fst
	 */
	public void setId_fst(String Id_fst) {
		setAttrVal("Id_fst", Id_fst);
	}
	/**
	 * 一级分类名称
	 * @return String
	 */
	public String getFst_name() {
		return ((String) getAttrVal("Fst_name"));
	}
	/**
	 * 一级分类名称
	 * @param Fst_name
	 */
	public void setFst_name(String Fst_name) {
		setAttrVal("Fst_name", Fst_name);
	}
	/**
	 * 缺陷表主键
	 * @return String
	 */
	public String getId_qa_flt() {
		return ((String) getAttrVal("Id_qa_flt"));
	}
	/**
	 * 缺陷表主键
	 * @param Id_qa_flt
	 */
	public void setId_qa_flt(String Id_qa_flt) {
		setAttrVal("Id_qa_flt", Id_qa_flt);
	}
	/**
	 * 整改说明
	 * @return String
	 */
	public String getRfm_des() {
		return ((String) getAttrVal("Rfm_des"));
	}
	/**
	 * 整改说明
	 * @param Rfm_des
	 */
	public void setRfm_des(String Rfm_des) {
		setAttrVal("Rfm_des", Rfm_des);
	}
}

File: client/iih_stub/iih.ci_stub/iih.ci.mrqc/itmdto/d/ItmFstDTO.cs
using System;
using xap.mw.core.data;
using xap.mw.coreitf.d;

namespace iih.ci.mrqc.itmdto.d
{
    /// <summary>
    /// ItmFstDTO 的摘要说明。
    /// </summary>
    public class ItmFstDTO : BaseDTO {

        public ItmFstDTO() {
 
        }

        /// <summary>
        /// 一级分类
        /// </summary>
		public string Id_fst {
            get { return getAttrVal<string>("Id_fst",null); }
            set { setAttrVal<string>("Id_fst", value); }
        }

        /// <summary>
        /// 一级分类名称
        /// </summary>
		public string Fst_name {
            get { return getAttrVal<string>("Fst_name",null); }
            set { setAttrVal<string>("Fst_name", value); }
        }

        /// <summary>
        /// 缺陷表主键
        /// </summary>
		public string Id_qa_flt {
            get { return getAttrVal<string>("Id_qa_flt",null); }
            set { setAttrVal<string>("Id_qa_flt", value); }
        }

        /// <summary>
        /// 整改说明
        /// </summary>
		public string Rfm_des {
            get { return getAttrVal<string>("Rfm_des",null); }
            set { setAttrVal<string>("Rfm_des", value); }
        }
        /// <summary>
        /// 返回属性列表
        /// </summary>
        /// <returns></returns>
        public override string[] getAttrNames()
        {
            return new string[]{ "Id_fst", "Fst_name", "Id_qa_flt", "Rfm_des"};
        }
        
        /// <summary>
        /// 返回全路径DO类名
        /// </summary>
        /// <returns></returns>
        public override string getFullClassName()
        {
            return "iih.ci.mrqc.itmdto.d.ItmFstDTO";
        }
    }
}
